### PR TITLE
AKS baseline - upgrade kubernetes version to v 1.21.x

### PR DESCRIPTION
**Pull request by magrande**
Tested using RG:  rg-bu0001a0008-k

### CODE REVIEW

**Comment by ckittel** [comment]
https://github.com/mspnp/aks-secure-baseline/blob/cdef9878e0d247522b50b79e32be3928c225f32d/README.md#L27

If you could update the README to be accurate as well, that would be appreciated.
image

File: README.md [hunk 1]
```diff
@@ -1,159 +1,159 @@
...
-- [Microsoft Azure Well-Architected Framework](https://docs.microsoft.com/azure/architecture/framework/)
-- [Microservices architecture on AKS](https://docs.microsoft.com/azure/architecture/reference-architectures/containers/aks-microservices/aks-microservices)
-
-## Contributions
-
-Please see our [contributor guide](./CONTRIBUTING.md).
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact <opencode@microsoft.com> with any additional questions or comments.
-
-With :heart: from Microsoft Patterns & Practices, [Azure Architecture Center](https://aka.ms/architecture).
```
Comment (ckittel) [inline README.md:159]: Looks like you changed line endings again.  Any reason?  If you have a local configuration thing going on, you can make this edit right in the GitHub UI, that won't cause line ending changes, IIRC.

Comment (magrande) [reply README.md]: Yes, sorry for that... fixed

**Review by ckittel — APPROVED** [review]
🚢 :)

**Pull request merged**